### DISCUSSION TITLE
AgGrid : Add call_column_api_method in same way as call_api_method

### DISCUSSION THREAD

**Discussion by aersam** [zauberzeug/nicegui#1267] [Ideas / Feature Requests]
There is a really useful method called `call_api_method` on Ag Grid which is defined as : 
```javascript
    call_api_method(name, ...args) {
      this.gridOptions.api[name](...args);
    },
```

however, ag grid has a number of very common methods on the columnApi Property. Would be really nice to have a `call_column_api_method` function that would simply be:
```javascript
    call_column_api_method(name, ...args) {
      this.gridOptions.columnApi[name](...args);
    },
```

This would allow to simple toggle column visibility, set widths etc with the need for run_javascript

could also do a PR, it's seems to be so simple to add :)

**Comment by falkoschindler** [comment]
Sounds great, @aersam!
A PR would be very welcome. Ideally it could include a little demo showing off this new feature. 😎

**Reply by aersam** [reply]
Here we go: https://github.com/zauberzeug/nicegui/pull/1359
